packages/@tinacms/cli/src/cmds/start-server/index.ts

```diff
@@ -46,13 +52,90 @@ interface Options {
 
 const gqlPackageFile = require.resolve('@tinacms/graphql')
 
+const resolveGitRoot = async () => {
+  const pathParts = process.cwd().split(path.sep)
+  while (true) {
+    const pathToGit = pathParts.join(path.sep)
+    if (await fs.pathExists(path.join(pathToGit, '.git'))) {
+      return pathToGit
+    }
+
+    if (!pathParts.length) {
+      throw new Error(
+        'Unable to locate your .git folder (required for isomorphicGitBridge)'
+      )
+    }
+    pathParts.pop()
+  }
+}
+
+async function makeIsomorphicOptions(fsBridge: FilesystemBridge) {
+  const gitRoot = await resolveGitRoot()
+  const options = {
+    gitRoot,
+    authorName: '',
+    authorEmail: '',
+    onPut: async (filepath: string, data: string) => {
+      await fsBridge.put(filepath, data)
```

Comment:
What does this actually do since we're already committing the change to this file?